firmware/applications/SignBuddy/nucleo_board.h

```diff
@@ -21,9 +21,9 @@
 #define SYSCLK_FREQ               24000000
 
 /* LED */
-#define LED1_PORT                 GPIOC
-#define LED1_PIN                  LL_GPIO_PIN_13
-#define GPIO_LED_CLK_EN()         LL_IOP_GRP1_EnableClock(LL_IOP_GRP1_PERIPH_GPIOC)
+#define LED1_PORT                 GPIOA
+#define LED1_PIN                  LL_GPIO_PIN_8
+#define GPIO_LED_CLK_EN()         LL_IOP_GRP1_EnableClock(LL_IOP_GRP1_PERIPH_GPIOA)
```

Comment:
PA5

Reply:
how do we handle what is already on pa5?

Reply:
ah forget about that

Reply:
maybe for the nucleo board we can disable some of the touch sensors